### PR TITLE
[Console] Add command for showing available Sylius plugins

### PR DESCRIPTION
**Pull request by GSadee**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | no
| New feature?    | no?
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

zrzut ekranu 2018-12-5 o 12 04 49

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Command/ShowAvailablePluginsCommand.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $output->writeln('<comment>Available official Sylius Plugins:</comment>');
+
+        $pluginTable = new TableRenderer($output);
+
+        $pluginTable->setHeaders(['Plugin', 'Description', 'URL']);
+        $pluginTable->addRow(['<info>Admin Order Creation</info>', 'Creating (and copying) orders in the administration panel.', 'https://github.com/Sylius/AdminOrderCreationPlugin']);
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Command/ShowAvailablePluginsCommand.php]: IMHO a list of plugins with whole data as an array and for loop with this function would be nicer solution. Even better if we add VO :)

Comment (Zales0123) [reply src/Sylius/Bundle/CoreBundle/Command/ShowAvailablePluginsCommand.php]: VO `PluginInfo(string $name, string $description, string $url)` would be great 👌 but an array of data would also work well :)

File: src/Sylius/Behat/Resources/config/suites.yml [hunk 2]
```diff
@@ -1,11 +1,13 @@
 # This file is part of the Sylius package.
+# This file is part of the Sylius package.
```
Comment (Zales0123) [inline src/Sylius/Behat/Resources/config/suites.yml]: Duplicated :)

File: src/Sylius/Behat/Context/Cli/ShowingAvailablePluginsContext.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    {
+        $this->application = new Application($this->kernel);
+        $this->application->add(new SetupCommand());
+
+        $this->command = $this->application->find('sylius:show-available-plugins');
+        $this->tester = new CommandTester($this->command);
+
+        try {
+            $this->tester->execute(['command' => 'sylius:show-available-plugins']);
+        } catch (\Exception $e) {
```
Comment (Zales0123) [inline src/Sylius/Behat/Context/Cli/ShowingAvailablePluginsContext.php]: Should we silence the exception? And `$e` -> `$exception` :)

**Comment by bartoszpietrzak1994** [comment]
@stefandoorn We will consider it in further iterations :)

**Comment by bartoszpietrzak1994** [comment]
Thanks, Grzegorz! :1st_place_medal:

**Pull request merged**